### ISSUE TITLE
Delete Projects

### ISSUE THREAD

**Issue by knoepsche** [SuperFlyTV/SuperConductor#209]
**No possibility to delete projects**
If you create projects then you can choose them by the "Open Existing Project" button. But there is no possibility to delete one. So this list gets longer and longer. Even after uninstall and reinstall the list stays the same.

**A further button to delete a project**
TTo my opinion the best would be a delete button next to the "Open" Button in the list where to choose a project. 

**Is there another way to delete the project data?**
After an uninstall the deletion of "user/AppData/Roaming/superconductor" didn't kill the project data as I exptected to. Is there a recommended way to bring the app into its default state?

**Comment by nytamin** [comment]
Yeah, there currently is no delete button for projects, we should add that..

On Windows, SuperConductor stores its data files at C:/Users/YourUser/Documents/SuperConductor, so if you delete the contents of that folder, you'll clear all settings and projects.
